uint16 positionStatus
uint16 orientationStatus
float64 latitude
float64 longitude
float32 altitude
float32 roll
float32 pitch
float32 yaw
float32 velX
float32 velY
float32 velZ
float32 accX
float32 accY
float32 accZ
float32 rateX
float32 rateY
float32 rateZ